nixos/modules/services/networking/tailscale.nix

```
@@ -41,15 +48,15 @@ in {
   };
 
   config = mkIf cfg.enable {
-    warnings = optional (firewallOn && rpfIsStrict) "Strict reverse path filtering breaks Tailscale exit node use and some subnet routing setups. Consider setting `networking.firewall.checkReversePath` = 'loose'";
+    warnings = optional (firewallOn && rpfIsStrict && !cfg.ignoreReversePathFilter) "Strict reverse path filtering breaks Tailscale exit node use and some subnet routing setups. Consider setting `networking.firewall.checkReversePath` = 'loose'";
```

Might be worth mentioning the option to silence it here?
```suggestion
    warnings = optional (firewallOn && rpfIsStrict && !cfg.ignoreReversePathFilter) "Strict reverse path filtering breaks Tailscale exit node use and some subnet routing setups. Consider setting `networking.firewall.checkReversePath` = 'loose', or silence this warning with `services.tailscale.ignoreReversePathFilter = true;`";
```

---

great suggestion! thanks and done